server/src/services/Google.ts

```diff
@@ -231,3 +236,21 @@ export async function deleteCalendarEvent({
     }),
   );
 }
+
+export async function testTokens() {
+  const apis = await createCalendarApis();
+
+  await Promise.all(
+    apis.map(async ({ tokenId, createApi }) => {
+      const calendarApi = await createApi();
```

Comment:
```suggestion
    apis.map(async ({ tokenId, calendarApi }) => {
```
The refactoring in `initGoogle` lets us write a simpler loop here.